### ISSUE TITLE
[Bug] Accessibility, screenreader ignores or skips items in nested stacklayout

### ISSUE THREAD

**Issue by rnic99** [xamarin/Xamarin.Forms#8613]
### Sample 
https://github.com/xamarin/xamarin-forms-samples/tree/master/UserInterface/Accessibility

### Description
Encountered an issue on an app I was working on where our custom entry fields were being skipped when swiping over them when using a screen reader. Its almost as if these items were put at the bottom of the accessibility tree, because if you continue to navigate down the page in question eventually you will be navigated to the controls that were skipped. Given It was a custom control I thought it was something on our end, but I was able to reproduce the issue on a Xamarin forms sample app. This issue occurs on both Android and iOS however their behavior is different from each other.

### Steps to Reproduce
1. Load the Accessibility sample app/Turn on Screen reader.  
2. Navigate to the app and click on the automation properties button .
3. While on the accessibility demo page swipe from left to right to navigate down the page. 
4. On Android the ("Enter your name label, and Enter name here entry field)" are skipped)/  iOS your are stopped from navigating further into the page at the ( Enter name here entry field). 

### Expected Behavior
Screen readers should be able to navigate from left to right and then down unless a tab index has been set or the item in question has been removed from the accessibility tree. 

### Actual Behavior
Screen readers are skipping or stopping at certain elements within nested stack layouts.
### Basic Information

- Version with issue: 4.0 and up 
- Last known good version: 3.6 
- IDE: VS for mac
- iOS:  13.1.2, 12.1.3
- Android: 10, 9
- Affected Devices: Pixel 2 XL, OnePlus 3T, Ipad (6th gen), iPhone 6.

### Screenshots
Android
One swipe on Android skips the ui elements

![Screenshot_20191121-100056](https://user-images.githubusercontent.com/55463610/69349805-6b8f0300-0c46-11ea-9282-ffe090091081.png)

![Screenshot_20191121-100105](https://user-images.githubusercontent.com/55463610/69349807-6d58c680-0c46-11ea-9516-f1cae94b6cea.png)
 
iOS

Adding AutomationProperties.IsInAccessibleTree="true" fixes the issue somewhat for iOS, this allows you to navigate past the entry field however it groups it together. If you continue to swipe down the page it will eventually move back to the fields individually. 

![IMG_0011](https://user-images.githubusercontent.com/55463610/69348529-54e7ac80-0c44-11ea-8ec5-e4272e259383.PNG)

This is how the screen reader handles the app without any changes I cannot navigate past the entry field.

![IMG_0012](https://user-images.githubusercontent.com/55463610/69348541-59ac6080-0c44-11ea-9c1b-582cf4b0921f.PNG)

### Additional Information 
Removing the stack layout surrounding the label and entry will resolve this issue on both Android and iOS.

**Comment by rnic99** [comment]
Has anyone had the chance to take a look at this issue yet?

**Comment by samhouts** [comment]
@rnic99 Can you confirm that this is still occurring on 4.4.0? Thanks!

**Comment by rnic99** [comment]
@samhouts Sure thing! I pulled down the latest stable which at this time is XF 4.4.0.991265, and can confirm that the issue is still persistent on both Android and iOS.

**Comment by samhouts** [comment]
@rnic99 Awesome! Well, not awesome that it's still broken, but thank you for taking the time to verify it for me. We'll get this prioritized. :)

**Comment by rnic99** [comment]
@samhouts No problem at all happy to do it, and thank you!

**Comment by Josh-Ng** [comment]
This is potentially related to an Issue I have been experiencing with Xamarin Forms 4.0+, For android, could you try creating a custom content page and overriding OnChildAdded(Element).
Then safe cast the element to a Visual Element and set IsTabStop to false. It looks like IsTabStop is defaulted to true and all TabIndexes are defaulted to 0 we can sometimes see unexpected behavior due to UpdateTraversalOrder in PageRenderer on android always overriding the default behavior as implemented. 

On iOS something similar happens (there is a protocol override which messes up the accessibility hierarchy), so you will need to override the PageRenderer's LoadView() method and return something else (can probably test by just returning a new UIView().

Though this may be a new, separate issue..
